scripts/exportFromSketch.js

```diff
@@ -42,37 +61,6 @@ exec(
 	}
 );
 
-/**
- * Uses sketchtoolUtils to export `fileNames` sketch files
- * to specified format and platform
- *
- * @param {Array} fileNames - list of modified files from SRC_DIR
- */
-const exportFiles = fileNames => {
```

Comment:
removed this function, as we can just do the forEach loop directly in the `exec` block below